src/django/pfb_analysis/views.py

```diff
@@ -96,6 +96,7 @@ class NeighborhoodBoundsGeoJsonViewList(APIView):
 
     pagination_class = None
     filter_class = None
+    permission_classes = (AllowAny,)
```

Comment:
`DjangoModelPermissionsOrAnonReadOnly` only works on views with a `queryset` defined. These are read-only endpoints just for rendering GeoJSON; the POST requests all go to the primary `Neighborhood` endpoint.